### ISSUE TITLE
When finishing a text selection outside contentidor it registers a click

### ISSUE THREAD

**Issue by YLivay** [gamejolt/next-issue-tracker#1299]
I'm not sure if that's just normal browser input behaviour or not, but it's especially bad when you're writing a comment in a modal and accidentally end your selection outside the content editor box. It just immediately closes the modal.

**Comment by TruddyTheDuddi** [comment]
I had this one too! It happened twice. I am not sure of the reason though. I clicked the backspace button, but I'm not sure if that is why it happened. Maybe there should also be something asking if you want to close the modal in case your text is longer than 50 characters maybe. (Would you like to discard the text, yes, no)

**Comment by YLivay** [comment]
Yeah we usually have this in place for forms by default. we just have to recreate this for the comments as a special case since they are not using the form components.
